### DISCUSSION TITLE
nuxt build failing when raw CSS file can not be loaded.

### DISCUSSION THREAD

**Discussion by Hyperblaster** [nuxt/nuxt#17231] [Questions]
trying to run `nuxt build` on my nuxt 3 application, however it fails when trying to build due to an external npm module, the npm module includes a css file and 

Line 1 of the 3rd party module's main javascript file is `import './index.css';`

I can confirm the file is there, it appears that either the vite/rollup builder is failing to load it, or the path isn't being match relative to the node module folder..
`

```
Rollup error: Could not load raw:./index.css (imported by node_modules/ui-library/dist/lib.es.js): ENOENT: no such file or directory, open './index.css' 
```

Any help or advice on what i may be doing wrong would be greatly appreciated.

**Comment by mirumirumi** [comment]
Hi!
I have encountered a similar problem, but at that time it worked fine by removing the comment-outs that were included in the css.
Do you have comments in that css ?
Please try it :)

**Comment by raishid** [comment]
I have the same error, could you solve it?

**Comment by gokaybiz** [comment]
Same issue..

**Comment by Echora-1** [comment]
> trying to run `nuxt build` on my nuxt 3 application, however it fails when trying to build due to an external npm module, the npm module includes a css file and
> 
> Line 1 of the 3rd party module's main javascript file is `import './index.css';`
> 
> I can confirm the file is there, it appears that either the vite/rollup builder is failing to load it, or the path isn't being match relative to the node module folder.. `
> 
> ```
> Rollup error: Could not load raw:./index.css (imported by node_modules/ui-library/dist/lib.es.js): ENOENT: no such file or directory, open './index.css' 
> ```
> 
> Any help or advice on what i may be doing wrong would be greatly appreciated.

HI! you solved the problem?

**Reply by mrserzhan** [reply]
Hey guys, I've faced with this issue as well, but I managed to solve it the same way as here: https://github.com/nuxt/nuxt/issues/22730

**Comment by Riyenz** [comment]
as @mrserzhan suggest, try the ff in nuxt.config.ts

```
  build: {
    transpile: ["ui-library"],
  },
```
